src/main.rs

```diff
@@ -101,7 +101,11 @@ fn load_config(options: &cli::Options) -> Config {
         });
 
     Config::load_from(&*config_path).unwrap_or_else(|err| {
-        error!("Error: {}; Loading default config", err);
+        match err {
+            ConfigError::NotFound => info!("Error: {}; Loading default config", err),
```

Comment:
My initial issue was probably a bit imprecise, but if a configuration file cannot be found, that should be an error.

However if the configuration file is empty, it should be only an `info!` log. So instead of using `ConfigError::NotFound` this should be changed into `ConfigError::Empty` instead.

Note that when the configuration file is not present at all, it will not raise the `ConfigError::NotFound` error. So this is specifically about a configuration file which can't be found after the path has been explicitly specified. So an error is appropriate for that.

Reply:
Ah, ok. I didn't know that `NotFound` is only raised when the config file is specified.

But if the file exists and is empty, doesn't that mean that it was explicitly created? To me it would make sense to not show the error only in the case that the default config file (like `$XDG_CONFIG_HOME/alacritty/alacritty.yml`) doesn't exist.

Reply:
The `Empty` error pops up when you run `alacritty --config-file /dev/null` for example. It's a good way to test things with the default config and shouldn't cause any issues.

> But if the file exists and is empty, doesn't that mean that it was explicitly created?
Exacly. So if the user explicitly creates an empty configuration file, that shouldn't cause an issue. An empty configuration file should just assume the defaults for every option should be used. And the defaults of every option are the default configuration file.

Reply:
Alright, I understand it now. I was thinking about the case where the default config file doesn't exist, but I think in that case alacritty generates one, right? I'll push another commit tommorow.

Reply:
Yeah, if the configuration file doesn't exist yet, Alacritty will generate it. That's unrelated to this issue.